.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'feature/issue-57_standalone-executable'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - name: "Create base filename for all artifacts"
+        id: basefn
+        shell: bash
+        run: |
+          FILEPATH=$(echo rctab_${{ github.ref_name }}_${{ runner.os }} | sed -e 's/\//_/g')
+          echo "FILEPATH=$FILEPATH" >> $GITHUB_OUTPUT
+
+      # Normalize platform-specific filepaths generated by gradle
+      - name: "Create .zip filename"
+        id: zipfn
+        shell: bash
+        run: echo "FILEPATH=build/${{ steps.basefn.outputs.FILEPATH }}.zip" >> $GITHUB_OUTPUT
+      - name: "Get extension"
+        id: ext
+        shell: bash
+        run: |
+          if [ ${{ runner.os }} == 'Windows' ]; then
+            echo "EXT=.exe" >> $GITHUB_OUTPUT
+          elif [ ${{ runner.os }} == 'Linux' ]; then
+            echo "EXT=" >> $GITHUB_OUTPUT
+          else
+            echo "EXT=.dmg" >> $GITHUB_OUTPUT
+          fi
+      - name: "Get jpackage output filepath"
+        id: jpackagefn
+        shell: bash
+        run: |
+          # TODO Issue-662 sync version number with Main.java and build.gradle
```

Comment:
Can you leave a comment about why these numbers need to be hard-coded in here? I have a feeling the question will come up repeatedly in the future.